src/journal/journalctl.c

```diff
@@ -842,6 +845,10 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_action = ACTION_SYNC;
                         break;
 
+                case ARG_OUTPUT_FIELDS:
+                        arg_output_fields = strv_split(optarg, ",");
```

Comment:
needs OOM check, i.e.
```c
if (!arg_output_fields)
        return log_oom();
```

Reply:
also, I think it would be nicer to permit multiple --output-fields= specifications, and the lists should be merged. i.e.

```
{
_cleanup_strv_free char **v = NULL;
v = strv_split(optarg, ",");
if (!v)
        return log_oom();
if (!arg_output_fields) {
        arg_output_fields = v;
        v = NULL;
} else {
        r = strv_extend_strv(&arg_output_fields, v, true);
        if (r < 0)
                 return log_oom();
}
```

Reply:
Seconded. In fact would it be better, perhaps, if a single option took only one value? I.e one option occurrence - one field name, repeat as necessary?

Reply:
The latter would work better with a single-character option name, though.

Reply:
@spbnick i'd prefer if we'd keep this similar in style to "systemctl show --property=", which also takes a comma separated list, and may be specified more than once. I think it's a good idea to remain internally consistent.

Reply:
@poettering Makes sense, agreed.

Reply:
Good idea. Changed it to @poettering's suggestion.